src/Codeception/Module/WebDriver.php

```diff
@@ -640,13 +640,16 @@ public function _savePageSource($filename)
      * $I->amOnPage('/user/edit');
      * $I->makeScreenshot('edit_page');
      * // saved to: tests/_output/debug/edit_page.png
+     * $I->makeScreenshot();
+     * // saved to: tests/_output/debug/2017-05-26_14-24-11.png
      * ?>
      * ```
      *
      * @param $name
      */
-    public function makeScreenshot($name)
+    public function makeScreenshot($name = null)
     {
+        $name = ($name) ? $name : date("Y-m-d_H-i-s");
```

Comment:
In case a test will be executed in parallel this may produce a conflict between 2 files.
how about adding `uniqid` to the name?

also please refactor this code using `if` operator.